commands/utils/set_args.js

```diff
@@ -227,6 +227,14 @@ function sync_args_from_cmd(args) {
     } else {
       lt_config["tunnel_settings"]["autostart"] = true;
     }
+   
+    if ("network" in args) {
+      lt_config["run_settings"]["network"] = true
+        ? args["network"] == "true"
+        : false;
+    } else if (!lt_config["run_settings"]["network"]) {
+      lt_config["run_settings"]["network"] = false;
```

Comment:
Please add validation to validate this in validate.js
that the value is either true or false

Reply:
resolved